index.html

```diff
@@ -1,12 +1,70 @@
+<!DOCTYPE html>
+<html>
 
+<head></head>
+<style>
+	.title {
+	text-align: center;
+	font-size: 100px;
+	font-family:  Arial, Times New Roman, sans-serif;
+	}
+	
+	
+	.explanation_1 {
+	position: absolute;
+	top: 200px;
+	left: 300px;
+	text-align: center;
+	height: 10%;
+	width: 50%;
+	font-size: 25px;
+	font-family: Arial, Times New Roman, sans-serif;
+	background-position: center;
+	background-color: gold;
+	margin: auto;
+	
+	}
+	
+	.explanation_2 {
+	position: absolute;
+	top: 350px;
+	left: 300px;
+	text-align: center;
+	background-color: blue;
+	}
+	
+	.whats_special {
+	position: absolute;
+	top: 450px;
+	left: 300px;
+	text-align: center;
+	background-color: green;
+	}
+	.footer {
+	position: absolute;
+	bottom: 50px;
+	right: 50px;
+	}
+</style>
+<body>
 
-<h1>Home</h1>
+    <h1 class = title>MATH WORLDS
+	</h1>
+	<p>
+	<div class = explanation_1>Math Worlds is a young nonprofit organization.</div>
+	
+	
+	<div class = explanation_2>The mission of Math Worlds  is to support every preschooler and elementary school student in developing perseverance, problem solving skills, and a love of mathematics. 
 
-
-<h2>Mission Statement</h2>
-
-<p>Math Worlds’ mission is to support elementary school children in developing perseverance and
-problem solving skills while learning mathematics.
-To achieve this, Math Worlds is developing a structured collection of online games and
-activities that will require minimal adults’ support and contain almost no explanations,
-thus letting children develop their math knowledge on their own in a highly individualized way.</p>
+	<br>
+	To achieve this, Math Worlds is developing a structured collection of online games and activities that will let children learn mathematics in a highly individualized, efficient and enjoyable way.
+	</div>
+	<div class = whats_special>
+	<a href = https://www.google.com>What is special about MW games?</a>
+	</div>
+	</p>
+    <p class = footer>
+		Site is being made by <a href = https://www.google.com>Eliezer Markov.</a>
```

Comment:
The link should point to something more specific than the Google front page :)

Reply:
That is just a placeholder as I'm actually not sure where Mrs. Dubinsky wants this to go.

Reply:
To something of your choosing I presume ;)